src/Jupyter/ConfigurationSource.cs

```diff
@@ -61,6 +61,9 @@ public interface IConfigurationSource
         /// </summary>
         public double TruncationThreshold =>
             GetOptionOrDefault("dump.truncationThreshold", 1e-10);
+
+        public PhaseDisplayStyle PhaseDisplayStyle =>
```

Comment:
It may be helpful to have a comment similar to the one above on `TruncateSmallAmplitudes` here as well.

Reply:
Looks good, thank you!